source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst

```diff
@@ -0,0 +1,32 @@
+.. rn:: 1.0.0
+
+Percona Kubernetes Operator for XtraDB Cluster
+==============================================
+
+Percona announces the general availability of |Percona Kubernetes Operator for XtraDB Cluster| 1.0.0 on May 29, 2019. This release is now the current GA release in the 1.0 series. `Install the Kubernetes Operator for PXC by following the instructions <https://www.percona.com/doc/kubernetes-operator-for-pxc/kubernetes.html>`__. Please see the `GA release announcement <https://www.percona.com/blog/2019/05/29/percona-kubernetes-operators/>`__. All of Percona's software is open-source and free.
```

Comment:
"Percona Kubernetes Operator for XtraDB Cluster" and "Kubernetes Operator for PXC"
both don't seem correct